numpy/typing/mypy_plugin.py

```diff
@@ -0,0 +1,62 @@
+"""A module containing `numpy`-specific plugins for mypy."""
+
+from __future__ import annotations
```

Comment:
To double check: We did drop support for python 3.6, right?

Reply:
The `pypy37` tests are still running on python 3.6, so I guess that's a no.
Removed as of https://github.com/numpy/numpy/pull/17843/commits/f85cebade4a68b16da448f3b71285cf20b3d115d.

Reply:
We dropped it, if there are still tests running, that is probably a small error, but if its no difference here...

Reply:
whoops, [this is wrong](https://github.com/numpy/numpy/blob/master/.github/workflows/build_test.yml#L187), it should be 
```
wget -q https://downloads.python.org/pypy/pypy3.7-v7.3.2-linux64.tar.bz2 -O pypy.tar.bz2
```
I will submit a Pr to fix this.